Issue 40082: Assay transform script errors

labkey-martyp opened the pull request:
Re-implement transform script html errors.  Adds errors written to errors.html into the Run Properties section of assay import.

labkey-martyp committed:
Issue 40082: Assay transform script errors

Re-implement transform script html errors.

labkey-jeckels reviewed (changes requested):
Changes look OK, but I recommend using org.labkey.api.util.DOM to build up the HTML instead. We're trying to use it for new code, and I think you could use it for this section without having to swap it in for existing code around it.

labkey-martyp committed:
Merge branch 'develop' into fb_40082_transform_script_errors

labkey-martyp committed:
Issue 40082: Update to use api.util.DOM

- Add font tag to api.util.DOM
- Update assay wizard transform error to use api.util.DOM
- Update .gitignore

labkey-martyp commented:
@labkey-jeckels Ok, I updated to use api.util.DOM.
